### DISCUSSION TITLE
新版本Carousel中的onSwipe失效了

### DISCUSSION THREAD

**Discussion by iiwhiiler** [ant-design/ant-design#49042] [Q&A]
之前的onSwipe是在滑动结束后，滑动后的卡与之前的卡不一样，才触发， 现在有的swipeEvent还能用，但是一滑动就触发

**Comment by yoyo837** [comment]
上个复现开issue？

**Reply by iiwhiiler** [reply]
好的。https://github.com/ant-design/ant-design/issues/49045
